### DISCUSSION TITLE
How to Transfer USDT Gasless

### DISCUSSION THREAD

**Discussion by sunsoftdev** [bcnmy/biconomy-client-sdk#513] [Q&A]
We have a use case where we need to move USDT tokens from a temporary wallet to a main wallet. For this, the gas fee needs to be sponsored, as the temporary wallet does not have the native token to cover the gas fees. Unfortunately, we couldn't find clear documentation on your site regarding USDT tokens for this specific use case.

Is there any sample or reference document for that. ?

**Comment by himanshugarg06** [comment]
is this temporary wallet  a smart account wallet? If yes then you can follow this script - https://github.com/bcnmy/sdk-examples/blob/master/scripts/gasless/erc20Transfer.ts
also this section is dedicated to the community calls, for general Q/A please start the conversation in [here](https://github.com/orgs/bcnmy/discussions/categories/q-a).

**Comment by sunsoftdev** [comment]
No the temporary wallet is not a Smart wallet it is EOA with a Private key, generated HD Wallet instance programmatically. How to use for such cases ?

**Comment by himanshugarg06** [comment]
It needs to be a smart wallet in order to make use of Bico infra for Account abstraction.

**Comment by subzz4u** [comment]
Hi @himanshugarg06, Can you please help me with this 
https://github.com/orgs/bcnmy/discussions/547
